spec.html

```diff
@@ -33428,9 +33428,12 @@ <h1>String.prototype.lastIndexOf ( _searchString_ [ , _position_ ] )</h1>
           1. If _numPos_ is *NaN*, let _pos_ be +&infin;; otherwise, let _pos_ be ! ToIntegerOrInfinity(_numPos_).
           1. Let _len_ be the length of _S_.
           1. Let _start_ be the result of clamping _pos_ between 0 and _len_.
+          1. If _searchStr_ is the empty String, return 𝔽(_start_).
           1. Let _searchLen_ be the length of _searchStr_.
-          1. Let _k_ be the largest possible non-negative integer not larger than _start_ such that _k_ + _searchLen_ &le; _len_, and for all non-negative integers _j_ such that _j_ &lt; _searchLen_, the code unit at index _k_ + _j_ within _S_ is the same as the code unit at index _j_ within _searchStr_; but if there is no such integer, let _k_ be -1.
-          1. Return 𝔽(_k_).
+          1. For each integer _i_ starting with _start_ such that _i_ &le; _len_ - _searchLen_, in descending order, do
```

Comment:
```suggestion
          1. For each non-negative integer _i_ starting with _start_ such that _i_ &le; _len_ - _searchLen_, in descending order, do
```

or

```suggestion
          1. For each integer _i_ starting with _start_ such that 0 &le; _i_ &le; _len_ - _searchLen_, in descending order, do
```

Reply:
went with "and" instead of a semicolon, lmk your thoughts

Reply:
Oh, I didn't mean to suggest both. I was okay with either, but both is unnecessary. And my latter suggestion used chained comparison, not a semicolon.

Reply:
oh hmm. i'll stick with the first suggestion then.